packages/naps2-deb/naps2-deb.pacscript

```diff
@@ -0,0 +1,15 @@
+name="naps2-deb"
+gives="naps2"
+depends=("libsane2")
+repology=("project: ${gives}")
+pkgver="7.4.1"
+arch=('arm64' 'amd64')
+if [[ ${CARCH} == "arm64" ]]; then
+  url="https://github.com/cyanfish/${gives}/releases/download/v${pkgver}/${gives}-${pkgver}-linux-amd64.deb"
```

Comment:
Seems like you've made a typo
```suggestion
  url="https://github.com/cyanfish/${gives}/releases/download/v${pkgver}/${gives}-${pkgver}-linux-arm64.deb"
```

Reply:
oh dang, yeah sorry, will commit suggestion